frontend/src/components/GameSettings/GameSettings.spec.js

```diff
@@ -84,6 +91,9 @@ describe('GameSettings', () => {
 
       state = {
         loading: ['getting-packs'],
+        players: {},
```

Comment:
I just realized this line is no longer needed because I'm now using the `playerIDs` array to get the amount of players.

Reply:
Them remove it?